backend/src/main/java/org/sefglobal/core/academix/service/SubCategoryService.java

```diff
@@ -139,6 +138,8 @@ public SubCategory updateSubCategory(long id, SubCategory subCategory)
             throw new ResourceNotFoundException(msg);
         }
         subCategory.setCategory(category.get());
+        subCategory.getTranslations()
+                .forEach(subCategoryTranslation -> subCategoryTranslation.setSubCategory(subCategory));
```

Comment:
I think we can resolve this by not linking the SubCategory in the SubCategoryTranslation class. WDYT?

Reply:
But how does it map when it is not linked?

Reply:
One side is enough to generate it I guess.

Reply:
But we can't remove it because we need it for the primary key

Reply:
oh I see